sos/plugins/openstack_heat.py

```diff
@@ -32,6 +34,18 @@ def setup(self):
             suggest_filename="heat_db_version"
         )
 
+        run_cmds = False
+        for os_var in ['OS_USERNAME', 'OS_PASSWORD', 'OS_TENANT_NAME']:
+            if os_var not in os.environ:
+                self.soslog.warning("%s not found in environment variables"
+                                    " which is required to run commands"
+                                    % (os_var))
+            else:
```

Comment:
Ditto as above. If the intention is to set run_cmds to True only if all three vars are set, then try e.g.:

    run_cmds = True
    for os_var ..:
        if os_var not in os.environ:
            run_cmds = False
            self.soslog.warning(..)